src/components/filter/filter-date/FilterDate.tsx

```diff
@@ -37,6 +41,27 @@ export const FilterDate = ({
 
   const [editingFilterText, setEditingFilterText] = useState<string>('');
 
+  // adjusts the date to set the filter to be inclusive of the entered value
+  const getAdjustedDate = (date: string, type: 'begin' | 'end') => {
+    switch (rangeType) {
+      case 'is between':
+      case 'is equal to':
+        if (type === 'begin') {
+          return dayjs(date).subtract(1, 'days').format(defaultDateFormat);
+        }
+        if (type === 'end') {
+          return dayjs(date).add(1, 'days').format(defaultDateFormat);
+        }
+        return date;
+      case 'is on or after':
+        return dayjs(date).subtract(1, 'days').format(defaultDateFormat);
+      case 'is before or on':
+        return dayjs(date).add(1, 'days').format(defaultDateFormat);
+      default:
+        return date;
+    }
+  };
```

Comment:
Looks like the return type for `dateBegin` and `dateEnd` in reducer is nullable, what do you think about making the date argument to be nullable so you don't have to have default nullish value to empty string?
Also, what do you think about make the arguments to props instead when we have more than 2 properties?
```suggestion
  const getAdjustedDate = ({date, type}: {date: string, type: 'begin' | 'end'}) => {
    switch (rangeType) {
      case 'is between':
      case 'is equal to':
        if (type === 'begin') {
          return dayjs(date).subtract(1, 'days').format(defaultDateFormat);
        }
        if (type === 'end') {
          return dayjs(date).add(1, 'days').format(defaultDateFormat);
        }
        return date;
      case 'is on or after':
        return dayjs(date).subtract(1, 'days').format(defaultDateFormat);
      case 'is before or on':
        return dayjs(date).add(1, 'days').format(defaultDateFormat);
      default:
        return date;
    }
  };
```